src/views/Hearing/Sidebar.js

```diff
@@ -29,6 +29,15 @@ class Sidebar extends React.Component {
         <div className="hearing-sidebar">
           <Row>
             <Col sm={6} md={12}>
+              <div className="sidebar-section commentNumber">
+                <Icon name="comment-o"/> {' '}
+                <FormattedMessage id="aTotalOf"/> {' '}
```

Comment:
Maybe this "Total X comments" thing should be a single message... Something like
```
<FormattedPlural value={hearing.n_comments}
    one={<FormattedMessage id="totalSubmittedComment" values={{n: hearing.n_comments}}/>}
    other={<FormattedMessage id="totalSubmittedComments" values={{n: hearing.n_comments}}/>}
/>
```
and
```
{
  "totalSubmittedComment": "Yhteensä {n} mielipide",
  "totalSubmittedComments": "Yhteensä {n} mielipidettä",
}
```

or however the exact syntax goes? :)